### ISSUE TITLE
Suggested edits for Ecology data carpentry - Introduction - Motivations for the OpenRefine Lesson

### ISSUE THREAD

**Issue by partiecolored** [datacarpentry/OpenRefine-ecology-lesson#198]
https://datacarpentry.org/OpenRefine-ecology-lesson/00-getting-started/index.html

I suggest removing the content below, and moving it to a more appropriate location:

"You must export your modified dataset to a new file: OpenRefine does not write back into your original sources. If you don’t save it, your OpenRefine work will be lost."

This isn't a motivation as much as an important reminder.

**Comment by deidrewhitmore** [comment]
Agreed that the current wording is either unclear or the statement is in the wrong location. If the intention is to emphasize that OpenRefine reduces the risk of saving over your raw data the last sentence ("If you don't save it, your OpenRefine work will be lost") should be removed. Additionally, the point could be more explicit- "You must export your modified dataset to a new file: OpenRefine does not write back into your original sources. The raw data file remains unchanged." or "You must export your modified dataset to a new file: OpenRefine does not save over the original source file. All changes are stored in the OpenRefine project."

**Comment by villanueval** [comment]
Wording changed implemented in the working version of the revision to the lesson: https://github.com/villanueval/OpenRefine-ecology-lesson/commit/a5dbfbe21d524d8bb8aed5a236130e3d7155e755
